contrib/README.md

```diff
@@ -0,0 +1,285 @@
+# Event Gateway on Kubernetes 
+
+This chart deploys the Event Gateway with etcd onto a Kubernetes cluster. Please note, the default instructions expect
+an existing kubernetes cluster that supports loadbalancers, such as GKE. If your environment doesn't have a loadbalancer
+set up, please follow the minikube instructions below to retrieve the `event_gateway` information.
+
+## Contents
+
+1. [Quickstart](#quickstart)
+1. [Minikube or local clusters](#minikube-or-local-clusters)
+    1. [Using helm](#using-helm)
+    1. [Using custom resources](#using-custom-resources)
+    1. [Setting up an Ingress](#setting-up-an-ingress)
+1. [Examples](#examples)
+1. [Configuration](#configuration)
+1. [Cleanup](#cleanup)
+
+## Quickstart
+
+Make sure you have helm installed on your machine and run `helm init` on your k8s cluster. This will set up the
+`helm` and `tiller` functions required for easy deployment of config files to your cluster. You can follow
+instructions [here](https://docs.helm.sh/using_helm/#quickstart) if you have not set this up previously.
+
+**NOTE:** This portion of the config expects you to have a pre-existing kubernetes cluster (not minikube). For 
+local development please check the [minikube](#minikube-or-local-clusters) information below.
+
+Once installed, navigate to the `event-gateway/contrib/helm` folder and install the following components:
+
+**etcd-operator**
+```bash
+helm install stable/etcd-operator --name ego [--namespace <namespace>]
+```
+
+**event-gateway**
+```bash
+helm install event-gateway --name eg [--namespace <namespace>]
+```
+
+This will install each of the `etcd-operator` and `event-gateway` into the `default` namespace in kubernetes. Please note,
+this namespace has no bearing on your Event Gateway `spaces` as outlined in the [docs](https://github.com/serverless/event-gateway/blob/master/README.md). If you'd like to install `etcd-operator` and `event-gateway` in another namespace, add the `--namespace <namespace>` option to both `helm install` commands above.
+
+Next we'll need to collect the Event Gateway IP and ports for use on the CLI. To do so, inspect your services as follows:
+
+```bash
+export EVENT_GATEWAY_URL=$(kubectl get svc event-gateway -o jsonpath={.status.loadBalancer.ingress[0].ip})
+export EVENT_GATEWAY_CONFIG_API_PORT=4001
+export EVENT_GATEWAY_EVENTS_API_PORT=4000
+```
+
+With your environment set up, you can now jump to the [examples](#examples) section to put your `event-gateway` to use!
+
+## Minikube or local clusters
```

Comment:
I think we should simply link to this page https://kubernetes.io/docs/tasks/tools/install-minikube/

Reply:
Also lates version of Docker includes simply way to run K8s locally https://docs.docker.com/docker-for-mac/kubernetes/#override-the-default-orchestrator. I think we should also include that in this guide.

Reply:
I would disagree with your first comment, as the `k8s` documentation is notoriously difficult to read. I was debating where to include the raw `fedora` instructions but opted to do it ourselves because each environment is different. Still pending instructions on the MacOS side, but will update that shortly.

ACK on the second comment, will investigate this option soon.

Reply:
Regarding installation guide, so maybe better options is to extract it to separate md file?  Probably most of the people interested in k8s will have minikube already installed. I think it will be better this guide will be straight-forward.

Reply:
Good call on that, I'll move the instructions to a supplemental file and link to it here. Will ping you when that update is ready.